media/web-dictaphone/scripts/app.js

```diff
@@ -1,24 +1,20 @@
-// set up basic variables for app
-
+// Set up basic variables for app
 const record = document.querySelector('.record');
 const stop = document.querySelector('.stop');
 const soundClips = document.querySelector('.sound-clips');
 const canvas = document.querySelector('.visualizer');
 const mainSection = document.querySelector('.main-controls');
 
-// disable stop button while not recording
-
+// Disable stop button while not recording
 stop.disabled = true;
 
-// visualiser setup - create web audio api context and canvas
-
+// Visualiser setup - create web audio api context and canvas
 let audioCtx;
 const canvasCtx = canvas.getContext("2d");
 
-//main block for doing the audio recording
-
+// Main block for doing the audio recording
 if (navigator.mediaDevices.getUserMedia) {
-  console.log('getUserMedia supported.');
+  console.log('The mediaDevices.getUserMedia() media is supported.');
```

Comment:
This comment is harder to understand than the one it's replacing. Should it be:

```suggestion
  console.log('The mediaDevices.getUserMedia() method is supported.');
```

? Also I wonder why we have single-quotes here, has this been Prettified?

Reply:
I have not prettified it. Let's add a commit for this, then run Prettifier (I'm so used of Prettifier run automatically)